package.json

```diff
@@ -34,6 +34,7 @@
   },
   "devDependencies": {
     "@alephium/sdk": "0.5.0",
+    "@alephium/tokens-list": "file:../tokens-meta/alephium-tokens-list-0.0.0.tgz",
```

Comment:
The package is not yet published and is in a private GH repo

Reply:
Where can I get this tarball?

Reply:
you can build it by checking out the master branch of the project https://github.com/alephium/tokens-meta and running

```
nvm use
npm i
npm run build
npm pack
```

Reply:
Actually, you don't have to do that anymore! I published the package and updated this PR.